pyuvdata/tests/test_ms.py

```diff
@@ -0,0 +1,110 @@
+"""Tests for MS object."""
+import nose.tools as nt
+import os
+from pyuvdata import UVData
+import glob as glob
+from pyuvdata.data import DATA_PATH
+import pyuvdata.tests as uvtest
+import numpy as np
+
+def test_readNRAO():
+    """Test reading in a CASA tutorial ms file."""
+    UV = UVData()
+    testfile=os.path.join(DATA_PATH,'day2_TDEM0003_10s_norx_1src_1spw.ms')
+    expected_extra_keywords=['data_column','antenna_positions']
+
+    uvtest.checkWarnings(UV.read_ms,
+                         [testfile],
+                         message='Telescope EVLA is not',
+                         nwarnings=0)
+    nt.assert_equal(expected_extra_keywords.sort(),
+                    UV.extra_keywords.keys().sort())
+    del(UV)
+
+    
+def test_noSPW():
+    """Test reading in a PAPER ms convertes by CASA from a uvfits with no spw axis."""
+    UV=UVData()
+    testfile_no_spw=os.path.join(DATA_PATH,'zen.2456865.60537.xy.uvcRREAAM.ms')
+    uvtest.checkWarnings(UV.read_ms,[testfile_no_spw],
+                         nwarnings=1)
+    del(UV)
+
+#!!!This test does not seem to work because importuvfits did not appear to preserve
+#!!!multiple subarray values going from .uvfits -> .ms
+#
+#def test_breakReadMS():
+#    UV=UVData()
+#    multi_subarray_file=os.path.join(DATA_PATH,'multi_subarray.ms')
+#    nt.assert_raises(ValueError,UV.read_ms,multi_subarray_file)
+#    del(UV)
+
+#Need a method to test casacore import error!
```

Comment:
@anguta can you please add this as an issue and describe the problem for future investigation.  We don't need to solve every problem before we merge, but at least document them